### PR TITLE
LRDOCS-4424 using Liferay Util taglib

### PR DESCRIPTION
**Pull request by mwilliams2014**
@jbalsas I've ported the Util Taglib documentation to 7.1 and made a few updates. Please review and let me know if you have any doubts. Thanks!

### CODE REVIEW

**Comment by mwilliams2014** [comment]
@jbalsas Can you review this, this week?

File: develop/tutorials/articles/210-front-end-taglibs/02-liferay-util-taglibs/00-using-liferay-util-taglibs-intro.markdown [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+To use the Liferay-Util taglib, add the following declaration to your JSP:
+
+    <%@ taglib prefix="liferay-util" uri="http://liferay.com/tld/util" %>
+
+The Liferay-Util taglib is also available via a macro for your FreeMarker theme 
+templates and web content templates. Follow this syntax:
+
+    <@liferay_util["tag-name"] attribute="string value" attribute=10 />
+
+Each Liferay Util tag has an optional `outputKey` attribute. If several portlets
```
Comment (jbalsas) [inline develop/tutorials/articles/210-front-end-taglibs/02-liferay-util-taglibs/00-using-liferay-util-taglibs-intro.markdown]: This is not true at all. Only `body-bottom`, `body-top`, `html-bottom` and `html-top` have that as it's specific to their behaviour

File: develop/tutorials/articles/210-front-end-taglibs/02-liferay-util-taglibs/02-liferay-util-body-top.markdown [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+The body top tag is not a self-closing tag. The content placed between the 
+opening and closing of this tag is moved to the top of the `body` tag. When 
+something is passed using this taglib, the 
+[body_top.jsp](https://github.com/liferay/liferay-portal/blob/7.1.x/portal-web/docroot/html/common/themes/body_top.jsp#L25-L31) 
+is passed markup and outputs in this JSP. 
+
+The example configuration below uses the `<liferay-util:body-top>` tag to 
+include JavaScript provided by the portlet's bundle:
+
+    <liferay-util:body-top outputKey="contenttop" >
```
Comment (jbalsas) [inline develop/tutorials/articles/210-front-end-taglibs/02-liferay-util-taglibs/02-liferay-util-body-top.markdown]: Shouldn't `outputKey` should be `bodytop` for consistency with the one on top?

File: develop/tutorials/articles/210-front-end-taglibs/02-liferay-util-taglibs/06-liferay-util-html-bottom.markdown [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+The HTML bottom tag is not a self-closing tag. Content placed between the 
+opening and closing of this tag is moved to the bottom of the `<html>` tag. When 
+something is passed using this taglib, the 
+[bottom.jsp](https://github.com/liferay/liferay-portal/blob/master/portal-web/docroot/html/common/themes/bottom.jsp#L53-L59) 
+is passed markup and outputs in this JSP. 
+
+The example configuration below uses the `<liferay-util:html-bottom>` tag to 
+include JavaScript (a common use case) provided by the portlet's bundle:
+    
+    <liferay-util:html-bottom outputKey="addscript">
```
Comment (jbalsas) [inline develop/tutorials/articles/210-front-end-taglibs/02-liferay-util-taglibs/06-liferay-util-html-bottom.markdown]: Shouldn't `outputKey` be `htmlbottom` for consistency with the others?

File: develop/tutorials/articles/210-front-end-taglibs/02-liferay-util-taglibs/07-liferay-util-html-top.markdown [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+The HTML top tag is not a self-closing tag. The content placed between the 
+opening and closing of this tag is moved to the `<head>` tag. When something is 
+passed using this taglib, the 
+[top_head.jsp](https://github.com/liferay/liferay-portal/blob/master/portal-web/docroot/html/common/themes/top_head.jsp#L147-L153) 
+is passed markup and outputs in this JSP. 
+
+The example configuration below uses the `<liferay-util:html-top>` tag to 
+include additional CSS styles provided by the portlet's bundle:
+
+    <liferay-util:html-top>
```
Comment (jbalsas) [inline develop/tutorials/articles/210-front-end-taglibs/02-liferay-util-taglibs/07-liferay-util-html-top.markdown]: Missing `outputKey` in here...

File: develop/tutorials/articles/210-front-end-taglibs/02-liferay-util-taglibs/00-using-liferay-util-taglibs-intro.markdown (hunk 1 above)
Comment (mwilliams2014) [reply develop/tutorials/articles/210-front-end-taglibs/02-liferay-util-taglibs/00-using-liferay-util-taglibs-intro.markdown]: hmmm... thanks for letting me know!

File: develop/tutorials/articles/210-front-end-taglibs/02-liferay-util-taglibs/02-liferay-util-body-top.markdown (hunk 2 above)
Comment (mwilliams2014) [reply develop/tutorials/articles/210-front-end-taglibs/02-liferay-util-taglibs/02-liferay-util-body-top.markdown]: Yep. That makes sense. Thanks!

File: develop/tutorials/articles/210-front-end-taglibs/02-liferay-util-taglibs/06-liferay-util-html-bottom.markdown (hunk 3 above)
Comment (mwilliams2014) [reply develop/tutorials/articles/210-front-end-taglibs/02-liferay-util-taglibs/06-liferay-util-html-bottom.markdown]: 👍

File: develop/tutorials/articles/210-front-end-taglibs/02-liferay-util-taglibs/07-liferay-util-html-top.markdown (hunk 4 above)
Comment (mwilliams2014) [reply develop/tutorials/articles/210-front-end-taglibs/02-liferay-util-taglibs/07-liferay-util-html-top.markdown]: 👍

**Comment by mwilliams2014** [comment]
Thanks for the feedback @jbalsas ! I'll make the changes and send it forward.

**Pull request closed without merging**